services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/[asset]/index.js

```diff
@@ -125,7 +125,7 @@ const Asset = ({ libraryData, overviewMdxSource, params }) => {
           <PageBreadcrumb items={breadcrumbItems} />
         </Column>
         <Column sm={4} md={8} lg={{ start: 5, span: 12 }} className={styles['tabs-column']}>
-          <PageTabs className={styles['asset-tabs']} tabs={pageTabs} />
+          <PageTabs className={styles['asset-tabs']} tabs={pageTabs} title="Asset tabs" />
```

Comment:
I saw in the other comment between you and Matt that this is an aria thing and would potentially be announced by a screen reader. If that's the case, to a visually impaired user, it would be more useful for this to say "Page tabs" to them than it would for it to say "Asset tabs". It's a small distinction, but "Page tabs" as a way to describe "the tabs on a page" and the ability to switch amongst them is generally a familiar concept, whereas "Asset tabs" as a way to describe "the tabs on the page" is not.

Reply:
Done!